EventFilter/SiPixelRawToDigi/interface/PixelDataFormatter.h

```diff
@@ -107,6 +107,8 @@ class PixelDataFormatter {
   int maxROCIndex;
   bool phase1;
 
+  friend class SiPixelRawToDigi;
```

Comment:
.. actually, looking at the additions in SiPixelRawToDigi.cc,
it seems to be better to have all of the code lines there that convert SiPixelRawDataError to a PixelFEDChannel to be added as a new method here, e.g ```PixelFEDChannel error25Channel(const SiPixelRawDataError&) const ```.